pages/common/waitress-serve.md

```diff
@@ -1,6 +1,7 @@
 # waitress-serve
 
 > Pure Python WSGI HTTP Server.
+> More information: <https://docs.pylonsproject.org/projects/waitress/en/stable/runner.html>.
```

Comment:
```suggestion
> More information: <https://github.com/Pylons/waitress>.
```

Reply:
The project's website is <https://docs.pylonsproject.org/projects/waitress>. but the PR's author used a link to the pseudo man page

Reply:
@CleanMachine1 The project's website is [docs.pylonsproject.org/projects/waitress](https://docs.pylonsproject.org/projects/waitress). but the PR's author used a link to the pseudo man page

Reply:
OK, either works for me!

Reply:
@CleanMachine1 so, do you dismiss your suggestion?